firefoxThemes/popup/popup.css

```diff
@@ -122,6 +147,13 @@ input:checked + .slider:before {
   transform: translateX(26px);
 }
 
+/*Switch labels*/
```

Comment:
Could you move this to `popup.js` please and use this?

```javascript
label[for="backgroundType"],
label[for="darkMode"],
label[for="hideSearch"]
{
  color:${colors.infoForeground};
}
```

I like `infoForeground` because its themed and not as impactful as `foregroundColor` but I'd be willing to consider `foregroundColor` for usability.

Reply:
I agree `foregroundColor` seems to be more in tune as a fallback coloration.